examples/Atlas/sdf/cinder_block_wide/model.sdf

```diff
@@ -17,104 +17,23 @@
       </inertial>
```

Comment:
I will rebase this branch to remove the changes to the terrain (which are now part of master) and make sure it doesn't conflict